### PR TITLE
Update Schema_Overview.md

### PR DESCRIPTION
**Pull request by ctSkennerton**
Add in example that shows how to return a taxonomy string of all species in the database.

### CODE REVIEW

File: wiki/Schema_Overview.md [hunk 1]
```diff
@@ -250,6 +250,28 @@ nodes but lists the downstream, child path instead):
     WHERE name LIKE '%fungi%')
 ```
 
+Return the full taxonomy of every species in the database as a
+semicolon separated list. This query is written for postgres
+and relies on the `string_agg` function. SQLite and MySQL instead
+call the same function `group_concat`.
+
+``` sql
```
Comment (hlapp) [inline wiki/Schema_Overview.md:258]: It looks like you have a space between the backticks and the language identifier. Is that on purpose? All code block syntax I can find has this without the space.

**Review by hlapp — COMMENTED** [review]
Aside from the one inline comment looks good!

File: wiki/Schema_Overview.md (hunk 1 above)
Comment (ctSkennerton) [reply wiki/Schema_Overview.md:258]: every other code block in this file has the space, so I was just being consistent.

Comment (hlapp) [reply wiki/Schema_Overview.md:258]: OK - I'll merge and if there's a problem with the space I'll fix it wholesale.

Comment (peterjc) [reply wiki/Schema_Overview.md:258]: GitHub at least doesn't mind the space - I ran into this with a colleague's markdown. Not sure which is considered canonical/preferred.

**Pull request merged**